README.md

```diff
@@ -57,14 +57,40 @@ aws sqs create-queue --queue-name DemoApplicationQueue
 
 ## How to run
 
+The demo application can be run in two modes: `with-issues` and `without-issues`. Thus, you can compare how an
+optimized version of the application compares to one with performance issues.
+
+* To run the `with-issues` version, use the following instructions. Remember to replace `YOUR-ACCOUNT-ID`, `YOUR-AWS-REGION` and `YOUR-BUCKET-REPLACE-ME` as appropriate.
+
 ```bash
-export DEMO_APP_SQS_URL="https://eu-west-1.queue.amazonaws.com/YOUR-ACCOUNT-ID/DemoApplicationQueue"
-export DEMO_APP_BUCKET_NAME="demo-application-test-bucket-1092734-REPLACE-ME"
+# These need to be set according to your setup:
+export DEMO_APP_SQS_URL=https://YOUR-AWS-REGION.queue.amazonaws.com/YOUR-ACCOUNT-ID/DemoApplicationQueue
```

Comment:
The SQS URL is in form of https://sqs.us-east-1.amazonaws.com/YOUR-ACCOUNT-ID/DemoApplicationQueue.